controller/src/main/java/io/pravega/controller/store/stream/PravegaTablesStreamMetadataStore.java

```diff
@@ -131,6 +133,12 @@ public OperationContext createStreamContext(String scopeName, String streamName,
         return new StreamOperationContext(scope, stream, requestId);
     }
 
+    @Override
+    public <T> CompletableFuture<T> addEntryToDeletingScope(String scope, OperationContext context, ScheduledExecutorService executor) {
+        storeHelper.addNewEntry(DELETING_SCOPE_NAME, scope, UUID.randomUUID(), UUID_TO_BYTES_FUNCTION, context.getRequestId());
```

Comment:
Why are we adding a random UUID to this table? Shouldn;t this be UUID of the scope we're deleting?

Reply:
we are adding a new entry to the deleting table and hence adding a new random UUID